src/Sulu/Component/Content/Document/Subscriber/ResourceSegmentSubscriber.php

```diff
@@ -77,7 +89,7 @@ public function handleHydrate(AbstractMappingEvent $event)
 
         $node = $event->getNode();
         $property = $this->getResourceSegmentProperty($document);
-        $originalLocale = $this->inspector->getOriginalLocale($document);
+        $originalLocale = $this->documentInspector->getOriginalLocale($document);
```

Comment:
When this is integrated with the multiple document managers change, then this inspector will be sourced from the event.